src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -266,7 +268,7 @@ def test_webapp_config(self, resource_group):
             JMESPathCheck('pythonVersion', ''),
             JMESPathCheck('use32BitWorkerProcess', True),
             JMESPathCheck('webSocketsEnabled', False),
-            JMESPathCheck('minTlsVersion', '1.0')])
+            JMESPathCheck('minTlsVersion', '1.2')])
```

Comment:
@panchagnula, can you please confirm that the default TLS has been changed from 1.0 to 1.2?

Reply:
sure let me take a look

Reply:
![image](https://user-images.githubusercontent.com/13839372/43808656-dbd98830-9a62-11e8-96ad-c820d80dd5bf.png)
yes looks like this did change for newly created apps